### PR TITLE
chore: update readme file

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes # https://vsf.atlassian.net/jira/software/c/projects/SFUI2/boards/69?modal=detail&selectedIssue=SFUI2-1139&assignee=5fcf86fca1d3f100684db8a5

# Scope of work

Added section How to run project locally

# Screenshots of visual changes

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,13 @@ You can become contributor not only by writing code:
 
 We are grateful for all kind of contributions. If you don't know where to start check out our [contribution guide](./CONTRIBUTING.md) or ask for help on [Discord](https://discord.vuestorefront.io)
 
+## How to run project locally
+
+In the root directory of this project run `yarn` command to install all needed dependencies. 
+After dependencies has been installed run `yarn dev` command to run this project.
+Now both (Nuxt preview for Vue and Next preview for React) projects are running locally and you can start working on them.
+If you need to run only one of them, you need to run a desired preview from its directory, so you need to go for Vue: `/apps/preview/nuxt` and for React: `/apps/preview/next`. 
+
```
Comment (Szymon-dziewonski) [inline README.md:87]:
```suggestion
## How to run project locally

In the root directory of this project run `yarn` command to install all needed dependencies. 
After dependencies has been installed run `yarn dev` command to run this project.
Now both (Nuxt preview for Vue and Next preview for React) projects are running locally and you can start working on them.
If you need to run only one of them, you need to run a desired preview from its directory, so you need to go for Vue: `/apps/preview/nuxt` and for React: `/apps/preview/next`. 

NOTE:
Subdependencies have to be build, such as 
- `@storefront-ui/shared` located in `/packages/sfui/shared` 
- `@storefront-ui/tw-plugin-peer-next` located in `/packages/sfui/tw-plugin-peer-next` 
- `@storefront-ui/typography` located in `/packages/sfui/typography` 

Knowing this we recommend using `yarn dev` in `root` directory.
```

Comment (justyna-13) [reply README.md]: https://github.com/vuestorefront/storefront-ui/pull/2686/commits/96413df2e5c0a8d31cbe1f91b8f004ce753589bb

File: README.md [hunk 2]
```diff
@@ -80,11 +80,19 @@ We are grateful for all kind of contributions. If you don't know where to start
...
 If you need to run only one of them, you need to run a desired preview from its directory, so you need to go for Vue: `/apps/preview/nuxt` and for React: `/apps/preview/next`. 
 
+NOTE:
+Subdependencies have to be build, such as 
+- `@storefront-ui/shared` located in `/packages/sfui/shared` 
+- `@storefront-ui/tw-plugin-peer-next` located in `/packages/sfui/tw-plugin-peer-next` 
+- `@storefront-ui/typography` located in `/packages/sfui/typography` 
+- 
+Knowing this we recommend using `yarn dev` in `root` directory.
+
```
Comment (Szymon-dziewonski) [inline README.md:95]:
```suggestion
Subdependencies have to be build, such as 
- `@storefront-ui/shared` located in `/packages/sfui/shared` 
- `@storefront-ui/tw-plugin-peer-next` located in `/packages/sfui/tw-plugin-peer-next` 
- `@storefront-ui/typography` located in `/packages/sfui/typography` 

Knowing this we recommend using `yarn dev` in `root` directory.

```
Sorry mad mistake, one dash too much :(

Comment (justyna-13) [reply README.md]: https://github.com/vuestorefront/storefront-ui/pull/2686/commits

**Pull request merged**